test/contracts/supplementary/lib/exchangeIssuanceLibraryMock.spec.ts

```diff
@@ -231,13 +276,21 @@ contract('ExchangeIssuanceLibraryMock', accounts => {
       await subject();
     });
 
-    describe('when send quantities quantities is zero', async () => {
-      before(async () => {
-        subjectSendTokenAmounts = [ZERO];
+    describe('when send tokens is empty', async () => {
```

Comment:
Shouldn't there be more validations here around exchangeIds since this function checks that?